homeassistant/components/broadlink/helpers.py

```diff
@@ -1,5 +1,8 @@
 """Helper functions for the Broadlink integration."""
 from base64 import b64decode
+import socket
+
+import psutil
```

Comment:
Where are we depending on this requirement?

Reply:
We are using this dependency to determine IPv4/L3 broadcast addresses in a platform-agnostic way. I could implement our own logic in the library with the stdlib:
```python3
import subprocess
subprocess.check_output("ip addr show".split())
```
But this is OS-dependent. We don't have `ip addr show` on Windows for example, so I would need to add more logic to the code. I feel that anything different from psutils or netifaces is reinventing the wheel.

We don't have any of these dependencies in the library (I think they won't accept), but we have psutil in Home Assistant. How about calling `psutil.net_if_addrs()` only once, during startup, and consult the addresses in memory at each discovery?

Reply:
We don't have psutil in home assistant as a core requirement. It it's needed it needs to be added as a requirement in the manifest.